src/entities/day.ts

```diff
@@ -82,15 +86,15 @@ export const fullDayMap: Partial<Record<Day, FullDayJa>> = {
   AnyTime: "随時",
 };
 
-export const weekdayNum = (day: Day): number => weekdays.indexOf(day);
+export const weekdayNum = (day: WeekDay): number => weekdays.indexOf(day);
 export const weekNum = (day: Day): number => week.indexOf(day);
 
-export const weekDayToJa = (day: Day): WeekDayJa => weekdayMap[day] ?? "月";
+// 消したい
+export const dayToWeekJa = (day: Day): WeekDayJa => weekdayMap[day] ?? "月";
 export const dayToJa = (day: Day): DayJa => weekMap[day] ?? "月";
+export const dayToFullDayja = (day: Day): FullDayJa => fullDayMap[day] ?? "月";
 
-export const jaToDay = (ja: DayJa): Day =>
-  week.find((key) => weekMap[key] === ja) ?? "Unknown";
-export const scheduleJaToDay = (ja: ScheduleDayJa): Day =>
-  week.find((key) => weekMap[key] === ja) ?? "Unknown";
-export const fullDayJaToDay = (ja: FullDayJa): Day =>
-  fullDays.find((key) => fullDayMap[key] === ja) ?? "Unknown";
+/** 任意の値をCourseDayに変換できるように */
+export const jaToDay = (ja: string): CourseDay =>
```

Comment:
```suggestion
export const jaToDay = (ja: FullDayJa | NotWeekDay | "指定なし"): CourseDay =>
```
これで通ります。`FullDayJa | NotWeekDay | "指定なし"`の意味は 「entity で想定しているモジュールの日本語表記」という意味です。任意の `string` は型安全ではないので、今後新しいモジュールの日本語表記が出現したら`entiriy` にそれを追加した上で、使用するのが安全かと思います。
 なお`FullDayJa | NotWeekDay | "指定なし"`は ScheduleDay と等価ですが、ここに `SchduleDays`と書くのは違和感があったためこうしました。
`ExceptDayJa` `AllDayJa` のような「entity で想定しているモジュールの日本語表記」のような型があってもいいかもしれません。

Reply:
>  `string`だと型安全ではない


というのはもっともな意見なのですが、ここではapiで定義されている値に一対一で対応付けられるような値だけの変換を想定しています。`その他`などの値は使う場所によって意味合いが変わってくることがあるため、現状はこの関数を使う前に別途処理を行えばいいと思いますし、実際にそうしています。（`その他`や`指定なし`が引数に入ってくることはありません）`string`のところは後日変更すればいいと思うので一旦このまま行きたいです。（早くこのPRをmergeさせたい）

Reply:
> その他などの値は使う場所によって意味合いが変わってくることがあるため、現状はこの関数を使う前に別途処理を行えばいいと思いますし、実際にそうしています。

これはどういう意味ですか？結局ここで型安全を担保するぐらいだったら上記の変更をするのとああまり変わりがないと思うのですが

Reply:
電話で直接話し合って
> apiで定義されている値に一対一で対応付けられるような値だけの変換を想定しています。
な用途だからここの引き数は FullDayJa にしようという話になりました。

Reply:
`Array.filter`で`undefined`の可能性があり、`?? "Unknown"`は必要

Reply:
そこ実はたしかうまく型安全にする方法はあるのだけど、とりあえず引き数が`FullDayJa`なだけ安全なので Approve しました！